lib/shopify_cli/identity_auth.rb

```diff
@@ -195,30 +205,37 @@ def refresh_exchange_tokens
 
     def request_exchange_tokens
       APPLICATION_SCOPES.each do |key, scopes|
-        request_exchange_token(key, client_id_for_application(key), scopes)
+        request_and_save_exchange_token(key, client_id_for_application(key), scopes)
       end
     end
 
-    def request_exchange_token(name, audience, additional_scopes)
+    def request_and_save_exchange_token(name, audience, additional_scopes)
       return if name == "shopify" && !store.exists?(:shop)
+      access_token = request_exchange_token(
+        name: name,
+        audience: audience,
+        additional_scopes: additional_scopes,
+        subject_token: store.get(:identity_access_token),
+        destination: (name == "shopify") ? "https://#{store.get(:shop)}/admin" : nil
+      )
+      store.set("#{name}_exchange_token".to_sym => access_token)
+      ctx.debug("#{name}_exchange_token: " + access_token)
+    end
 
+    def request_exchange_token(name:, audience:, additional_scopes:, subject_token:, destination: nil)
       params = {
```

Comment:
@leleabhinav are these the right parameters to pass when exchanging the partners' token to generate one that we can use from the CLI?